linked_list/linkedList.js

```diff
@@ -0,0 +1,263 @@
+class Node {
+	// Constructs Node
+	constructor (data) {
+		this.data = data;
+		this.next = null;
+	}
+}
+
+class LinkedList {
+	// Initialises Linked List
+	constructor (size = 0) {
+		this.head = null;
+		this._size = size;
+	}
+
+	// Inserts Element with value at index
+	add (value, index) {
+		if (index < 0 || index > this._size) {
```

Comment:
If a user is setting size of a LinkedList, I believe it is same as setting capacity. So LinkedList should not contain more than `size` elements at any point of time. (The same happens in a normal C array, right? `int arr[5]`)
If size is set to 0, then LinkedList should become of infinite size.